### ISSUE TITLE
Were stft loss and waveform loss used in your training?

### ISSUE THREAD

**Issue by Aria-K-Alethia** [descriptinc/descript-audio-codec#17]
Hi,

Thank you for sharing your contribution.

I read the code and I found that you used stft loss and waveform loss (L1 recon loss), but based on the config params in `conf/base.yml`, it seems that these two loss have no weight, so I think they will not be used in the back-propagation.

I wonder if you used these two loss in your released models?
Do you think these loss are not useful for the training?

Thank you.

**Comment by pseeth** [comment]
We did not use these losses during training. Only multi-scale mel loss and the discriminators were needed. When the waveform loss weight is very low, we found it didn't really do much, and when it was set very high, we found it resulted in buzzy or fuzzy output. 

Similarly, multi-scale STFT loss didn't have much improvement. 

Something to note is that both of these losses are sorta baked into the discriminators, as there is a feature matching loss. The feature matching loss on the early layers of the discriminators inherently captures waveform and STFT loss.
